src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -46,10 +46,49 @@
 
 helps['consumption pricesheet'] = """
     type: group
-    short-summary: Inspect the price sheet by Azure subscription within current or specified billing period.
+    short-summary: Inspect the price sheet by Azure subscription within a billing period.
 """
 
 helps['consumption pricesheet show'] = """
     type: command
-    short-summary: Show the price sheet for Azure subscription within a current billing period or specified billing period
+    short-summary: Show the price sheet for Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace'] = """
+    type: group
+    short-summary: Inspect the marketplace by Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace list'] = """
+    type: command
+    short-summary: Show the marketplace for Azure subscription within a billing period.
+"""
+
+helps['consumption budget'] = """
+    type: group
+    short-summary: Offers ability to create, get, update, and delete budgets for Azure subscription.
```

Comment:
@sptramer do you have an opinion on this?  It seems like the more common way would be to say "Manage budgets for an Azure subscription."

Reply:
Already have a change on it, "Manage budgets for an Azure subscription"